src/main/java/duke/DukeList.java

```diff
@@ -0,0 +1,110 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import duke.task.Task;
+
+/**
+ * A list of tasks.
+ */
+public class DukeList implements Iterable<Task> {
+    /** ArrayList used to store tasks. */
+    private final ArrayList<Task> list;
+
+    /**
+     * Public constructor to create an empty DukeList.
+     */
+    public DukeList() {
+        list = new ArrayList<>();
+    }
+
+    /**
+     * Public constructor that takes in an
+     * ArrayList to create a DukeList.
+     *
+     * @param list ArrayList to copy tasks.
+     */
+    public DukeList(ArrayList<Task> list) {
+        this.list = list;
+    }
+
+    /**
+     * Returns the task at a specific index.
+     *
+     * @param i Index of the task.
+     * @return Task.
+     */
+    public Task get(int i) {
```

Comment:
Hi, overall you seemed to have followed the coding standards pretty well, so good job! Perhaps you could rename this method to getTask() to make it clearer what exactly is this method returning?